### PR TITLE
feat(i18n,esp):quality assurance cert fully translated and proofread

### PR DESCRIPTION
**Pull request by RafaelDavisH**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [ ] I have tested these changes either locally on my machine, or GitPod.

### CODE REVIEW

File: config/i18n/all-langs.ts [hunk 1]
```diff
@@ -45,7 +45,8 @@ export const auditedCerts = {
     SuperBlocks.JsAlgoDataStruct,
     SuperBlocks.FrontEndDevLibs,
     SuperBlocks.DataVis,
-    SuperBlocks.BackEndDevApis
+    SuperBlocks.BackEndDevApis,
+    SuperBlocks.QualityAssurance,
```
Comment (nhcarrigan) [inline config/i18n/all-langs.ts:49]:
```suggestion
    SuperBlocks.QualityAssurance
```
I think prettier is complaining about the trailing comma.

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]

File: config/i18n/all-langs.ts (hunk 1 above)
Comment (RafaelDavisH) [reply config/i18n/all-langs.ts]: Sorry, missed it.

**Review by naomi-lgbt — APPROVED** [review]
Looks good to me!

**Comment by sidemt** [comment]
LGTM 👍

**Review by raisedadead — APPROVED** [review]

**Pull request merged**